docs/for-developers.md

```diff
@@ -157,6 +162,7 @@ Exceptions should only be used in those cases when the application **must** stop
 
 As for memory allocations, spawning small dynamic objects is also discouraged. If possible, stack allocations are a better alternative. Arrays with variable length on stack are totally fine to be used in place of local objects allocated on heap.  
 `std::string` is also highly discouraged, make sure `std::string_view` is used instead whenever possible.
+<<<<<<< HEAD
```

Comment:
A leftover :D. 
I have been extending and reformatting the documentation a lot, so I just added your notice about docker to master so that you are not forced to handle too many collisions.